conftest.py

```diff
@@ -0,0 +1,16 @@
+# Configuration file for running our tests
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--env",
+        action="store",
+        required=False,
```

Comment:
This should either be required, or have a default. Having neither produces strange errors like `configparser.NoSectionError: No section: None`.